File: lingkaran.cpp
#include <iostream>
using namespace std;
int main() {
    int jarijari;
    cout<< "Masukkan jari-jari: " << endl;
    cin >> jarijari;
    float phi = 3.14;
    float luas = phi * jarijari * jarijari;
    float keliling = 2 * phi * jarijari;
    cout << "Keliling lingkaran adalah: " << keliling << endl;
    cout << "Luas lingkaran adalah: " << luas << endl;
    return 0;
}

File: lingkaran.py
print("============Menghitung Luas dan Keliling Lingkaran=============")

phi = 3.14
r = float(input("Masukkan jari-jari: "))

if r <= 0:
    print("Jari-jari tidak boleh negatif atau nol!")
else:
    luas = phi * r * r
    keliling = 2 * phi * r 
    
    print(f"Luas lingkaran adalah: {luas:.2f}")
    print(f"Keliling lingkaran adalah: {keliling:.2f}")